### ISSUE TITLE
Review Tom mechanics

### ISSUE THREAD

**Issue by Tom-van-Woudenberg** [TUDelft-CITG/pre-for-cem#42]
https://github.com/TUDelft-CITG/pre-for-cem/blob/8676bd510d332c3ee971a56b2e16826c53f0a0ae/book/contents/mechanics.md?plain=1#L18
- Source only includes loads with quite trivial information. Would be nice to have some formal definition of how schematizations are defined in general

**Comment by ahmedfarhatx22** [comment]
Dr. Roel is working on a reader that could have just that! Source needs to be updated once the reader is complete. (TASK NOT COMPLETE YET)
